libs/designsystem/src/lib/components/modal-experimental/fullscreen/fullscreen.component.html

```diff
@@ -0,0 +1,34 @@
+<ion-modal
+  [isOpen]="open"
+  [canDismiss]="canDismiss"
+  (willPresent)="_onWillPresent($event)"
+  (didPresent)="_onDidPresent($event)"
+  (willDismiss)="_onWillDismiss($event)"
+  (didDismiss)="_onDidDismiss($event)"
+>
+  <ng-template>
```

Comment:
Just out of curiosity - why is this wrapped in <ng-template>? 
I cant find the place where we are actually telling angular to render what is inside, what am I missing 😅

Reply:
Good question! :D It's required by the modal component. It's in the modal component's default slot, so I suspect that it handles the `<ng-template>` behind the scenes :D

Reply:
![image](https://media.tenor.com/rrLadwcIvTIAAAAM/unicorn-magic.gif)